File: development/OnBoarding/OnBoarding/OnBoarding.Contract/KnowledgeTransfer/KnowledgeTransfer.cs
namespace OnBoarding.Contract
{
    using System;

    public class KnowledgeTransfer
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public Guid ModeId { get; set; }

        public Guid? OwnerId { get; set; }

        public string Reference { get; set; }

        public decimal Duration { get; set; }         
    }
}

File: development/OnBoarding/OnBoarding/OnBoarding.Web/src/app/model/knowledgeTransfer.ts
export interface IKnowledgeTransfer {
    Id: string,
    Name: string,
    Description: string,
    Reference: string,
    Duration: number,
    ModeId: string,
    RoleId: string,
    OwnerId: string
}

export class KnowledgeTransfer implements IKnowledgeTransfer {

    constructor(
       public Id: string,      
       public Name: string,
       public Description: string,
       public Reference: string,
       public Duration: number,
       public ModeId: string,
       public RoleId: string,
       public OwnerId: string) {
    }
}